build.gradle

```diff
@@ -15,6 +15,7 @@ buildscript {
 plugins {
     id 'com.github.ben-manes.versions' version '0.17.0'
     id "org.sonarqube" version "2.5" apply false
+    id 'com.gradle.build-scan' version '1.16'
```

Comment:
I think you should be using build scan plugin `2.x` with Gradle 5.x. The latest build scan plugin version is `2.1`, so I suggest you go with that.